gnd/src/main/java/com/google/android/gnd/MainViewModel.java

```diff
@@ -42,7 +44,12 @@
 @SharedViewModel
 public class MainViewModel extends AbstractViewModel {
 
+  /** The window insets determined by the activity. */
+  @Hot(replays = true)
```

Comment:
Annotation already added to the public accessible method. Let's keep it at only one place.

Reply:
See previous comment.